Lib/test/libregrtest/runtest.py

```diff
@@ -206,6 +206,8 @@ def _test_module(the_module):
 def _runtest_inner2(ns, test_name):
     # Load the test function, run the test function, handle huntrleaks
     # and findleaks to detect leaks
+    minruns = sys._getopcacheminruns()
```

Comment:
I don't think that it matters to restore the old value. I'm not sure that sys._getopcacheminruns() is needed.